browser-test/src/application_navigation.test.ts

```diff
@@ -75,59 +77,229 @@ describe('Applicant navigation flow', () => {
-        'Sim',
-        'WA',
-        '54321',
-      )
-      await applicantQuestions.clickNext()
+        // Fill out address question and click next
+        await applicantQuestions.answerAddressQuestion(
+          '1234 St',
+          'Unit B',
+          'Sim',
+          'WA',
+          '54321',
+        )
+        await applicantQuestions.clickNext()
 
-      // Click previous and see file upload page with address
-      await applicantQuestions.clickPrevious()
-      await applicantQuestions.checkAddressQuestionValue(
-        '1234 St',
-        'Unit B',
-        'Sim',
-        'WA',
-        '54321',
-      )
+        // Click previous and see previous page with address
+        await applicantQuestions.clickPrevious()
+        await applicantQuestions.checkAddressQuestionValue(
+          '1234 St',
+          'Unit B',
+          'Sim',
+          'WA',
+          '54321',
+        )
 
-      // Click previous and see static question page
-      await applicantQuestions.clickPrevious()
-      await applicantQuestions.seeStaticQuestion('static question text')
+        // Click previous and see static question page
+        await applicantQuestions.clickPrevious()
+        await applicantQuestions.seeStaticQuestion('static question text')
 
-      // Click previous and see date and name questions
-      await applicantQuestions.clickPrevious()
-      await applicantQuestions.checkDateQuestionValue('2021-11-01')
-      await applicantQuestions.checkEmailQuestionValue('test1@gmail.com')
+        // Click previous and see date and name questions
+        await applicantQuestions.clickPrevious()
+        await applicantQuestions.checkDateQuestionValue('2021-11-01')
+        await applicantQuestions.checkEmailQuestionValue('test1@gmail.com')
 
-      // Assert that we're on the review page.
-      await applicantQuestions.clickPrevious()
-      await applicantQuestions.expectReviewPage()
+        // Assert that we're on the review page.
+        await applicantQuestions.clickPrevious()
+        await applicantQuestions.expectReviewPage()
+      })
+
+      it('clicking previous does not save when flag off', async () => {
+        const {page, applicantQuestions} = ctx
+        await loginAsAdmin(page)
+        await disableFeatureFlag(page, 'save_on_all_actions')
+        await logout(page)
+
+        await applicantQuestions.applyProgram(programName)
+        await applicantQuestions.answerDateQuestion('2021-11-01')
+        await applicantQuestions.answerEmailQuestion('test1@gmail.com')
+
+        await applicantQuestions.clickPrevious()
+
+        await applicantQuestions.expectReviewPage()
+        await applicantQuestions.validateNoPreviouslyAnsweredText(
+          dateQuestionText,
+        )
+        await applicantQuestions.validateNoPreviouslyAnsweredText(
+          emailQuestionText,
+        )
+      })
+
+      it('clicking previous with correct form shows previous page with saved answers', async () => {
```

Comment:
**question**: should this be "and saves answers"?

Reply:
Definitely, done.